src/main/resources/openapi/specs/api.kuflow.com/v1/openapi.yaml

```diff
@@ -265,6 +265,44 @@ paths:
         default:
           $ref: "#/components/responses/DefaultError"
 
+  /processes/{id}/~actions/save-user-action-value-document:
```

Comment:
I'm not sure, but maybe here we should use: `/processes/{id}/~actions/save-user-action-document:`

🤔

Reply:
I don't know, I left it like that for homogeneity with that of element...

Reply:
`save-element-value-document:`

Reply:
yap, but a user action can't have more than one value, isn't it?

Reply:
nop, only one value

Reply:
That's why I commented that 'value' is redundant, 🤔

Reply:
I'm not sure, but maybe is better left as is, as you have it, to avoid confusion.

Reply:
I don't know about the naming, but now is the right moment to make any change because later is a new API version :S